### DISCUSSION TITLE
Multiple active profiles for different architectures

### DISCUSSION THREAD

**Discussion by miquecg** [abiosoft/colima#609] [Q&A] [answered]
Hi,

I think it's currently possible to have profiles set for different architectures running containers at the same time. The idea would be to operate with `docker` within a different profile depending on the shell running the command.

I tried to achieve this by changing `docker context` but that seems to be global. In a sense it kind of works, because I could perfectly start containers on a shell and then change to another context on a different shell, start other services and all will be working I assume. The only problem is that I need to remember to change contexts back and forth.

Thanks.

**Comment by abiosoft** [comment, marked as answer]
You can have the `DOCKER_HOST` environment set for the current shell, it takes precedence over the context. 

The docker socket can be gotten from the output of `colima status`.

```sh
export DOCKER_HOST="unix://$HOME/.colima/default/docker.sock"
```

All docker commands will thereafter point to the profile.

**Reply by miquecg** [reply]
Thanks!
